example/Python/src/Piano/Piano.lf

```diff
@@ -0,0 +1,78 @@
+// Preambles are put inside the Python reactor class
+// Therefore, accessing them requires the use of self.
+target Python {
+    files: [sensor_simulator.py, keys.png, soundfont.sf2],
```

Comment:
The soundfont file is rather large and is really slowing down checkouts and fetches. We should probably use something like LFS (see: https://git-lfs.github.com/) in situations like these.